### PR TITLE
Sketcher: Fix copy paste errors

### PR DESCRIPTION
**Pull request by PaddleStroke**
Fix https://github.com/FreeCAD/FreeCAD/issues/15771
Fix 2 of the errors described in https://github.com/FreeCAD/FreeCAD/issues/15653

1 - Points being forced to construction when added to sketch (so on copy paste)
For this we simply have to remove a limitation that was here for unknown purpose. Apparantly to ensure legacy code would work (considered legacy 9 years ago).

2 - Driving and activated status of constraints not ported when copy pasted.

### CODE REVIEW

**Comment by PaddleStroke** [comment]
I think 
`Sketcher: addGeometry : Remove point construction limitation.`
Was making the tests fail. I just pushed without this commit to see if tests pass.

**Comment by chennes** [comment]
Still waiting on a review from me...

**Comment by FEA-eng** [comment]
I compiled this and it seems that _Sketcher_CreatePoint_ is working in the opposite way here. Currently, new points are added as normal geometry (same as other geometries) unless the construction mode is enabled. With this PR, new points are added as construction geometry by default and as normal geometry when the construction mode is enabled.

**Comment by yorikvanhavre** [comment]
Waiting for @chennes to review

**Comment by sliptonic** [comment]
@PaddleStroke  Can you take a look at the comment above be FEA-Eng?

**Comment by chennes** [comment]
I can reproduce @FEA-eng's reported problem: the sense of the driving-vs-reference selection is now inverted for points. On the plus side, copy and paste does keep them the same as they were.

**Comment by yorikvanhavre** [comment]
Still waiting for @PaddleStroke 's return

**Comment by PaddleStroke** [comment]
Bug fixed.

File: src/Mod/Sketcher/App/ConstraintPyImp.cpp
Comment (chennes) [inline src/Mod/Sketcher/App/ConstraintPyImp.cpp:71]: I don't love this abbreviation (since it matches the very-commonly-used std::string::cstr). I'd spell out "constraint" here.

**Review by chennes — CHANGES_REQUESTED** [review]
Looks good and works well, just a single quick rename suggestion.

File: src/Mod/Sketcher/App/ConstraintPyImp.cpp
Comment (PaddleStroke) [reply src/Mod/Sketcher/App/ConstraintPyImp.cpp]: Done

**Comment by yorikvanhavre** [comment]
Since @chennes 's last note is met, M3 considers this one good for merge

**Pull request merged**
